### DISCUSSION TITLE
How to add specific environment variable without prefix  NX_*, like NEXTAUTH_URL ?

### DISCUSSION THREAD

**Discussion by GreenHill19931222** [nrwl/nx#17236] [General]
According to NextAuth.js document , I need to add specific environment variable like ```NEXTAUTH_URL```  to solve this warning message ,
`[next-auth][warn][NEXTAUTH_URL]` 
`https://next-auth.js.org/warnings#nextauth_url`

However in Nx docs .env variable need to add prefix NX_* 
How could I set this enviroment variable in NX ?

**Comment by PhilipWee** [comment]
I am facing the same issue, and I've checked the nextauth source code and they only allow setting the variable via process.env in their source code

**Comment by moatorres** [comment]
You actually can use a `.env` inside the root of your project and it should be picked up automatically. Here's a live example from a repository of mine [https://github.com/codyslexia/nexa/tree/main/apps/frontend](https://github.com/codyslexia/nexa/tree/main/apps/frontend) that sets `NEXTAUTH_URL` (see the `.env.example` file).
